docs/resources/database.md

```diff
@@ -0,0 +1,69 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "oktapam_database Resource - terraform-provider-oktapam"
```

Comment:
We will want to use a template for resources with sensitive information that will be stored within the state file.  See https://github.com/hashicorp/terraform-provider-tls/blob/main/templates/resources/private_key.md.tmpl for an example of this and https://github.com/hashicorp/terraform-plugin-docs?tab=readme-ov-file#how-it-works for an explanation of the docs plugin.  

Also note that we're on a pretty old version of the docs plugin, so we may need to update it if we run into any issues with templating.

Reply:
Added a template but not entirely sure what all we want to say.  You probably saw the conversation a while back where I was trying to understand the best practice for people using this and how they'd provide a password... but I never quite got to the full answer.

Reply:
I also populated the template to be as similar to the auto generated one as possible (plus the addition of the Security Notice section)